### PR TITLE
support for execution of visualization object

### PR DESCRIPTION
**Pull request by peter-plochan**

### CODE REVIEW

File: src/main/java/com/gooddata/executeafm/ExecuteVisualizationService.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * Service for executeVisualization resource
+ */
+public class ExecuteVisualizationService extends ExecutionResultService {
+
+    private static final String EXECUTION_URI = "/gdc/app/projects/{projectId}/executeVisualization";
+
+    /**
+     * Service for executeVisualizationObject resource
```
Comment (liry) [inline src/main/java/com/gooddata/executeafm/ExecuteVisualizationService.java]: Try to write those comments from PoV of user who doesn't know GD resources into details ... it's not service for "resource" but for user, who wants to "do something" ;)

File: src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java [hunk 2]
```diff
@@ -27,12 +29,27 @@ public ExpressionFilter(@JsonProperty("value") final String value) {
         this.value = value;
     }
 
-   public String getValue() {
```
Comment (liry) [inline src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java]: typo?

File: src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java [hunk 3]
```diff
@@ -27,12 +29,27 @@ public ExpressionFilter(@JsonProperty("value") final String value) {
...
         return value;
     }
 
     @Override
     public String toString() {
         return GoodDataToStringBuilder.defaultToString(this);
     }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (liry) [inline src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java]: plz add also test using equalsverifier

File: src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java (hunk 2 above)
Comment (peter-plochan) [reply src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java]: nope, it's fix of missing space - methods were not aligned :)

Comment (liry) [reply src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java]: ohh, sorry, my bad %)

File: src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java (hunk 3 above)
Comment (peter-plochan) [reply src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java]: added

**Pull request merged**